aepp/configs.py

```diff
@@ -367,3 +370,141 @@ def setSandbox(self,sandbox:str=None)->dict:
+        Arguments
+            credentialId : OPTIONAL : The credential id that can be found on your Project page.
+            orgDevId : OPTIONAL : the org Id but NOT the IMS string. It is defined on your project page.
+                Example : https://developer.adobe.com/console/projects/<orgId>/<projectId>/credentials/<credentialId>/details/oauthservertoserver
+        """
+        if self.connectionType != "oauthV2":
+            raise Exception("You are trying to use a service that is only supportede for OauthV2 authen. We do not support the other auth types.")
+        if credentialId is None and self.credentialId is None:
+            raise ValueError("You are not providing the credential ID and did not use the setOauthV2setup method.\n Use it to prepare this method")
+        if orgDevId is None and self.orgDevId is None:
+            raise ValueError("You are not providing the orgDevId and did not use the setOauthV2setup method.\n Use it to prepare this method")
+        if credentialId is None and self.credentialId is not None:
+            credentialId = self.credentialId
+        if orgDevId is None and self.orgDevId is not None:
+            orgDevId = self.orgDevId
+        if self.token is None:
+            raise Exception("You need to generate a token by using the connect method first")
+        myheader = {
+            'Authorization' : 'Bearer '+self.token,
+            'x-api-key' : self.client_id
+        }
+        endpoint = f"https://api.adobe.io/console/organizations/{orgDevId}/credentials/{credentialId}/secrets"
+        res = self.connector.postData(endpoint,headers=myheader)
+        if 'client_secret' not in res.keys():
+            raise Exception("Could not find a client_secret in the key")
+        self.secret = res['client_secret']
+        self.__configObject__['secret'] = res['client_secret']
+        self.connector.config['secret'] = res['client_secret']
+        return res
+    
+    def updateConfigFile(self,destination:str=None)->None:
+        """
+        Once creating a client secret, you would need to update your config file with your new secret.
+        Returns True when successful
+        Arguments:
+            destination : REQUIRED : Destination path of the file name to updated.
+        """
+        if self.connectionType != 'OauthV2':
+            raise Exception('Do not support update for non Oauth Server to Server type')
+        json_data: dict = {
+            "org_id": self.org_id,
+            "client_id": self.client_id,
+            "secret": self.secret,
+            "sandbox-name": self.sandbox,
+            "scopes": self.scopes,
+            "environment": "prod"
+        }
+        with open(destination, "w") as cf:
+            cf.write(json.dumps(json_data, indent=4))
+        return True
+    
+    def deleteSecrete(self,secretUID:str=None,credentialId:str=None,orgDevId:str=None,)->None:
```

Comment:
lets rename to delete_secret

Reply:
As explained above, the package current naming convention is using lower camelCase, so I think we should keep it consistent. 
Otherwise, we would need to rename all existing methods which is a massive breaking change.